apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/McsTransformer.java

```diff
@@ -193,18 +191,19 @@ String convertIcn(Parser.Data<String> data) {
    */
   @VisibleForTesting
   void addDiagnosisCodes(McsClaim.Builder claimBuilder, Parser.Data<String> data, String icn) {
-    for (int i = 1; i <= MAX_DIAGNOSIS_CODES; ++i) {
+    for (int i = 0; i < Mcs.MAX_DIAGNOSIS_CODES; ++i) {
+      // We can't use the loop index directly because value must be final in the lambda expression
```

Comment:
Very helpful context comment, I was wondering about this as I was looking at it.